frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -568,6 +568,26 @@ function ReaderHighlight:addToMainMenu(menu_items)
             UIManager:show(items)
         end,
     })
+
+    if self.ui.rolling then
```

Comment:
Is it the first menu item in the **Settings** that is (not) shown depending on the type of the document?
On the contrary, we show "Save highlights to PDF" item when cre document is opened.

Reply:
> Is it the first menu item in the **Settings** that is (not) shown depending on the type of the document?

Yes, I think it is.

> On the contrary, we show "Save highlights to PDF" item when cre document is opened.

So, what are you suggesting ?
I guess it's not about moving `Save highlights` to this menu (which even if things are named Highlight... is more about text selection).
Are you suggesting always showing both, possibly making this one unchecked and disabled on non-CRE documents?

Reply:
No, I do not mean moving anything.

I think that general settings should be always visible and accessible.
(The first time I felt uncomfortable when met disabled "Scrolling" item in the Taps&Gestures menu).

Reply:
I think the disabled Scrolling is fine: if it wasn't, and scroll mode was off, the user would see the menu and its subitems, would tweak some of them, without any effect, and would think it's crap and doesn't work.
Showing it disabled, the user knows this doesn't apply currently, and would go looking for the other thing that would get it enabled. There is the much more similar behaviour in the bottom menu.

Same if we were showing fully enabled and actionable this "Auto-scroll in corner" on PDF: it would not work. Better let it be shown that it doesn't apply, and there's no need to allow the user to toggle stuff in a context where it doesn't apply and it can't verify what option is best for him.